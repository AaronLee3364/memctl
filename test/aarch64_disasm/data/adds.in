fffffff0070c0000 310005ce 2b080128 ab2a4109
