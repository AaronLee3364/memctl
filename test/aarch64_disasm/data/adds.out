fffffff0070c0000  ADDS    W14, W14, #0x1
fffffff0070c0004  ADDS    W8, W9, W8
fffffff0070c0008  ADDS    X9, X8, W10, UXTW
